secretmanager/api/SecretManagerSample/Program.cs

```diff
@@ -218,22 +224,24 @@ public static void CreateSecret(string projectId, string secretId)
 
         // [START secretmanager_delete_secret]
         /// <summary>
-        /// Delete an existing secret with the given name.
+        /// Delete an existing secret with the given name and etag (optional).
         /// </summary>
         /// <param name="projectId">ID of the project where the secret resides.</param>
         /// <param name="secretId">ID of the secret.</param>
+        /// <param name="etag">Current secret etag.</param>
         /// <example>
```

Comment:
I think we should have 2 examples for each of these. The existing example as-is with no etag and then another example (copy-paste the whole `<example>` block) that is "with etags". That will make it clearer that etags aren't required.